packages/inference/src/lib/makeRequestOptions.ts

```diff
@@ -29,7 +27,7 @@ export async function makeRequestOptions(
 	// eslint-disable-next-line @typescript-eslint/no-unused-vars
 	const { accessToken, model: _model, ...otherArgs } = args;
 	let { model } = args;
-	const { forceTask: task, includeCredentials, taskHint, ...otherOptions } = options ?? {};
+	const { forceTask: task, include_credentials, taskHint, ...otherOptions } = options ?? {};
```

Comment:
```suggestion
	const { forceTask: task, includeCredentials, taskHint, ...otherOptions } = options ?? {};
```
any reason for renaming?

Reply:
Just following the convention of these other ones:
https://github.com/huggingface/huggingface.js/blob/main/packages/inference/src/types.ts#L1-L26